core/src/com/unciv/logic/battle/Battle.kt

```diff
@@ -226,21 +222,76 @@ object Battle {
         }
     }
 
-    private fun tryCaptureUnit(attacker: MapUnitCombatant, defender: MapUnitCombatant): Boolean {
+    private fun tryCaptureUnit(attacker: ICombatant, defender: ICombatant, attackedTile: TileInfo): Boolean {
         // https://forums.civfanatics.com/threads/prize-ships-for-land-units.650196/
-        // https://civilization.fandom.com/wiki/Module:Data/Civ5/GK/Defines
+        // https://civilization.fandom.com/wiki/Module:Data/Civ5/GK/Defines\
+        // There are 3 ways of capturing a unit, we separate them for cleaner code but we also need to ensure a unit isn't captured twice
+
+        if (defender !is MapUnitCombatant || attacker !is MapUnitCombatant) return false
+
+        if (!defender.isDefeated() || defender.unit.isCivilian()) return false
+
+        fun unitCapturedPrizeShipsUnique():Boolean {
+            if (attacker.unit.getMatchingUniques(UniqueType.KillUnitCapture)
+                        .none { defender.matchesCategory(it.params[0]) }
+            ) return false
+
+            val captureChance = min(0.8f, 0.1f + attacker.getAttackingStrength().toFloat() / defender.getDefendingStrength().toFloat() * 0.4f)
+            return Random().nextFloat() <= captureChance
+        }
+
+        fun unitCapturedGermanyUnique():Boolean{
```

Comment:
I feel that `unitCapturedGermanyUnique()` should be renamed to something `unitGainFromEncampment` and `unitCapturedOttomanUnique()` renamed to something like `unitGainFromDefeatingUnit` since those are a touch more descriptive.